### ISSUE TITLE
Remove Limited

### ISSUE THREAD

**Issue by llorllale** [yegor256/cactoos#848]
`Limited` was deprecated in favor of `HeadOf` in #185. Let's delete it.

**Comment by 0crat** [comment]
The job [#848](https://github.com/yegor256/cactoos/issues/848) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/C63314D6Z/1291831516); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos#848` already assigned to @llorllale, can't assign to @llorllale

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by llorllale** [comment]
@0crat out

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
It is strongly discouraged to assign jobs to their creators, see [§19](http://www.zerocracy.com/policy.html#19): -15 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/cactoos/issues/848#issuecomment-387832900))

@llorllale GitHub issue is still open, won't close

**Comment by llorllale** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/cactoos/issues/848#issuecomment-387833853))

@llorllale This is what I know about this job, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#848](https://github.com/yegor256/cactoos/issues/848) is [in scope](http://www.zerocracy.com/policy.html#14) for 10min
  * The role is `DEV`
  * The job is assigned to @llorllale[/z](https://www.0crat.com/u/llorllale) for [10min](http://www.zerocracy.com/policy.html#8)
  * There is a monetary reward attached
  * @llorllale[/z](https://www.0crat.com/u/llorllale) will get 50% of their hourly rate for this job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @llorllale[/z](https://www.0crat.com/u/llorllale): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Ayegor256%2Fcactoos%23848%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by llorllale** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/cactoos/issues/848#issuecomment-387834288))

@llorllale The user @llorllale[/z](https://www.0crat.com/u/llorllale) resigned from [#848](https://github.com/yegor256/cactoos/issues/848), please stop working. Reason for job resignation: Order was cancelled

**Comment by llorllale** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/cactoos/issues/848#issuecomment-387835064))

@llorllale This is what I know about this job, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#848](https://github.com/yegor256/cactoos/issues/848) is [in scope](http://www.zerocracy.com/policy.html#14) for 14min
  * The role is `DEV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is not set yet
  * These users are banned and won't be assigned:
    * @llorllale[/z](https://www.0crat.com/u/llorllale): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Ayegor256%2Fcactoos%23848%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by llorllale** [comment]
@0crat out

**Comment by llorllale** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/cactoos/issues/848#issuecomment-387835792))

@llorllale This is what I know about this job, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#848](https://github.com/yegor256/cactoos/issues/848) is [in scope](http://www.zerocracy.com/policy.html#14) for 16min
  * The role is `DEV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is not set yet
  * These users are banned and won't be assigned:
    * @llorllale[/z](https://www.0crat.com/u/llorllale): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Ayegor256%2Fcactoos%23848%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/cactoos/issues/848#issuecomment-387835745))

@llorllale The job [#848](https://github.com/yegor256/cactoos/issues/848) is now out of scope

**Comment by llorllale** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/cactoos/issues/848#issuecomment-387837479))

@llorllale This is what I know about this job, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#848](https://github.com/yegor256/cactoos/issues/848) is not in scope
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Ayegor256%2Fcactoos%23848%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@38e2c9bb-8f57-4fb1-8558-17fff4f33af4, thread: AsyncFlush-1
com.zerocracy.farm.reactive.StkGroovy[125] java.lang.IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/in/orders/close_job.groovy
sun.reflect.NativeMethodAccessorImpl[-2] java.lang.reflect.InvocationTargetException: null
com.zerocracy.farm.strict.StrictProject[111] java.lang.IllegalArgumentException: File "blanks/llorllale.xml" is not accessible in "PMO"
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm): CID: 1291843919, Type: 'Close job', Author: llorllale

**Comment by llorllale** [comment]
Fixed in 85e57d79d84a1a209a00a584202801b42665196f

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@38e2c9bb-8f57-4fb1-8558-17fff4f33af4, thread: AsyncFlush-2
com.zerocracy.farm.reactive.StkGroovy[125] java.lang.IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/in/orders/close_job.groovy
sun.reflect.NativeMethodAccessorImpl[-2] java.lang.reflect.InvocationTargetException: null
com.zerocracy.farm.strict.StrictProject[111] java.lang.IllegalArgumentException: File "blanks/llorllale.xml" is not accessible in "PMO"
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm): CID: 1291852485, Type: 'Close job', Author: llorllale
